src/main/java/org/jabref/gui/preferences/journals/AbbreviationViewModel.java

```diff
@@ -94,4 +94,18 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(getName(), isPseudoAbbreviation());
     }
+
+    public boolean containsCaseIndependent(String s) {
+        s = s.toLowerCase();
```

Comment:
```suggestion
        s = s.toLowerCase(Locale.ROOT));
```
For case insenstive strings you should use it with Locale.ROOT